blueflood-http/src/main/java/com/rackspacecloud/blueflood/inputs/handlers/HttpMetricsIngestionServer.java

```diff
@@ -62,51 +60,32 @@
 
 public class HttpMetricsIngestionServer {
     private static final Logger log = LoggerFactory.getLogger(HttpMetricsIngestionServer.class);
-    private static TimeValue DEFAULT_TIMEOUT = new TimeValue(5, TimeUnit.SECONDS);
-    private static int WRITE_THREADS = Configuration.getInstance().getIntegerProperty(CoreConfig.METRICS_BATCH_WRITER_THREADS); // metrics will be batched into this many partitions.
-
+    private static TimeValue DEFAULT_TIMEOUT = new TimeValue(1, TimeUnit.MINUTES);
```

Comment:
@GeorgeJahad I think this is out of the scope of this PR. Returning 504 is going to change our API and we should discuss how we want to do it before making the change. 
As for the separate issue of changing the ingestion timeout, lets make it configurable as a first step and set it to current default value of 5 secs, if someone wants to change it, they will have to test it in their staging environment first to find the optimal setting.